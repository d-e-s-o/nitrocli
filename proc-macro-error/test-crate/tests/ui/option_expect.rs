extern crate test_crate;
use test_crate::make_fn;

make_fn!(option_expect);

fn main() {}
