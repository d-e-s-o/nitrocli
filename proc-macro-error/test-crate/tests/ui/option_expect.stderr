error: Option::expect_or_abort() test
 --> $DIR/option_expect.rs:4:1
  |
4 | make_fn!(option_expect);
  | ^^^^^^^^^^^^^^^^^^^^^^^^ in this macro invocation
